### DISCUSSION TITLE
is there method read file from classpath

### DISCUSSION THREAD

**Discussion by zhaoyta** [JanusGraph/janusgraph#3044] [Q&A]
i package "janusgraph-hbase.properties" in jar.
but when init .it is only init from file system.(in JanusGraphFactory.open)
```

val settings = new Settings
    settings.graphManager="org.janusgraph.graphdb.management.JanusGraphManager"
    settings.graphs =Collections.singletonMap("ConfigurationManagementGraph","janusgraph-hbase.properties")

    var gm = new JanusGraphManager(settings)
    GraphUtils.dropGraph(graphName)

```
can add JanusGraphManager(Configuration) or other support read from classpath.

janus version is v0.3.3

**Comment by li-boxuan** [comment]
Right now there is no such method. You could create a feature request issue if you want, but if you are packing your all jar anyway, you probably could modify JanusGraph source code [here](https://github.com/JanusGraph/janusgraph/blob/master/janusgraph-core/src/main/java/org/janusgraph/core/JanusGraphFactory.java).
